sapinvoices/config.py

```diff
@@ -31,3 +46,24 @@ def configure_sentry() -> str:
         sentry_sdk.init(sentry_dsn, environment=env)
         return f"Sentry DSN found, exceptions will be sent to Sentry with env={env}"
     return "No Sentry DSN found, exceptions will not be sent to Sentry"
+
+
+def load_config_values() -> dict:
+    settings = {}
+    for variable in CONFIG:
+        settings[variable] = os.getenv(variable)
+    # all settings are required in prod and stage
```

Comment:
Since most of the settings will be required in all envs, I'm not sure there's a particular reason not to just require them everywhere.